specs/darwin/install_history.table

```diff
@@ -0,0 +1,11 @@
+table_name("install_history")
```

Comment:
Let's prefix this with `package_` or rename to `package_history`. Then it's easier to discover/relate with `package_receipts` and `package_bom`.